src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
-'use strict';
+const succsesFunc = (message) => {
+  const successDiv = document.createElement('div');
 
-// write your code here
+  successDiv.className = 'message';
+  successDiv.innerText = message;
+  document.body.appendChild(successDiv);
+};
+
+const errorFunc = (error) => {
+  const errorDiv = document.createElement('div');
+
+  errorDiv.className = 'message error-message';
+  errorDiv.innerText = error;
+  document.body.appendChild(errorDiv);
+};
+
+document.addEventListener('DOMContentLoaded', () => {
+  const promise1 = new Promise((resolve, reject) => {
+    document.querySelector('.logo').addEventListener('click', () => {
+      resolve('Promise was resolved!');
+    });
+  });
+
+  const promise2 = new Promise((resolve, reject) => {
+    setTimeout(() => {
+      // eslint-disable-next-line prefer-promise-reject-errors
+      reject('Promise was rejected!');
```

Comment:
Be cautious with the use of `setTimeout` for rejecting promises. If the user clicks the logo before 3 seconds elapse, `promise1` will resolve, but `promise2` will still reject after 3 seconds. This could lead to a situation where both a success and error message are displayed, which might be confusing for the user.